Bottom: 40f5d398b06b210099a271fdbc139c05545bcee4
Top:    d56156c5496c38be22ecad15e02ff364021fd7bb
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-09 12:43:23 -0300

Refresh of template-support.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index 8bb413d..490285e 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -1636,7 +1636,7 @@ plugin_unary_type_expr (cc1_plugin::connection *self,
   plugin_context *ctx = static_cast<plugin_context *> (self);
   tree type = convert_in (operand);
   tree_code opcode = ERROR_MARK;
-  switch (CHARS2 (binary_op[0], binary_op[1]))
+  switch (CHARS2 (unary_op[0], unary_op[1]))
     {
       /* FIXME: implement sizeof, alignof, ...  */
     default:
